core/src/test/java/apoc/export/cypher/ExportCypherTest.java

```diff
@@ -1400,8 +1410,18 @@ public static class ExportCypherResults {
 
         public static final String EXPECTED_ONLY_SCHEMA_CYPHER_SHELL = convertToCypherShellFormat(EXPECTED_ONLY_SCHEMA_NEO4J_SHELL);
 
-        public static final String EXPECTED_NEO4J_SHELL_WITH_COMPOUND_CONSTRAINT = String.format("BEGIN%n" +
-                "CREATE CONSTRAINT PersonRequiresNamesConstraint FOR (node:Person) REQUIRE (node.name, node.surname) IS NODE KEY;%n" +
+        public static final List<String> EXPECTED_CONSTRAINTS = List.of(
```

Comment:
I'd rather move it to `ExportCypherEnterpriseFeaturesTest`, since it is only used there.
But otoh, here we have all `EXPECTED_` stuff. Up to you.

Reply:
Yes it seems like everything was defined here, but I agree that things that are only used in enterprise could be defined there instead. For consistency, I will not change it now, but feel free to make a Trello card about cleaning up all of them if you like.

Reply:
Added it: https://trello.com/c/ArNb37tZ/2192-move-attributes-from-exportcyphertestjava-to-exportcypherenterprisefeaturestestjava